scylla/src/transport/session_test.rs

```diff
@@ -33,6 +38,38 @@ use std::time::Duration;
 use tokio::net::TcpListener;
 use uuid::Uuid;
 
+#[cfg(not(feature = "cloud"))]
+type ConditionalSessionBuilderKind = DefaultMode;
+#[cfg(feature = "cloud")]
+type ConditionalSessionBuilderKind = CloudMode;
+
+/*
+  Creates a generic session builder based on conditional compilation configuration
+  For SessionBuilder of DefaultMode type, adds localhost to known hosts, as all of the tests
+  connect to localhost.
+*/
+fn create_new_session_builder() -> GenericSessionBuilder<ConditionalSessionBuilderKind> {
+    #[cfg(not(feature = "cloud"))]
+    {
+        let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+        let session_builder: GenericSessionBuilder<DefaultMode> =
+            SessionBuilder::new().known_node(uri);
+        session_builder
+    }
+
+    #[cfg(feature = "cloud")]
+    {
+        use crate::CloudSessionBuilder;
+        use std::path::Path;
+
+        let session_builder: GenericSessionBuilder<CloudMode> = std::env::var("CLOUD_CONFIG_PATH")
+            .map(|config_path| CloudSessionBuilder::new(Path::new(&config_path)))
+            .unwrap()
+            .unwrap();
```

Comment:
As this code is going to be called frequently (on each test), it might be helpful in debug to use `.expect(SomeMeaningfulMessage)` instead of plain `.unwrap()`.

Reply:
Fixed it, thanks.